apps/src/code-studio/components/libraries/LibraryPublisher.jsx

```diff
@@ -269,7 +271,7 @@ export default class LibraryPublisher extends React.Component {
             <input
               style={styles.largerCheckbox}
               type="checkbox"
-              id={functionName}
+              id={_.uniqueId(`${functionName}-`)}
```

Comment:
Typo: I think you missed updating the htmlFor in line 280 for this element.

Reply:
gah, great catch!!